lib/Doctrine/ORM/Mapping/Driver/AttributeReader.php

```diff
@@ -0,0 +1,87 @@
+<?php
+
+namespace Doctrine\ORM\Mapping\Driver;
+
+use Doctrine\Common\Annotations\Reader;
+use Doctrine\ORM\Mapping\Annotation;
+
+// TODO: Should we move this to doctrine/annotations?
```

Comment:
It doesn't look like this class has any hard relations to ORM at the moment, so it seems to be a good candidate for Annotations. Your improvements will have to coordinate with the Annotations repository and package.